source/stylesheets/pages/_home.scss

```diff
@@ -74,13 +77,17 @@ section.intro {
 // About
 
 section#about {
-  background: $accent-color;
-  @include grid-12;
-  transition: height .25s ease;
-  height: 0;
+  transition: all .5s ease;
   overflow: hidden;
-  &.slide-open {
-    height: 100%;
+  max-height: 0px;
+  padding: 0;
+  &.slide {
+    max-height: 100vh;
+  }
+  .wrapper {
+    background: $accent-color;
+    @include grid-12;
+    padding: 4%;
```

Comment:
This just adds that new `wrapper` div and migrates our grid layout from the parent section down to this level

Reply:
👍